pkgs/tools/wayland/wl-clipboard/default.nix

```
@@ -16,8 +23,8 @@ stdenv.mkDerivation rec {
   buildInputs = [ wayland ];
 
   meta = with lib; {
-    description = "Command-line copy/paste utilities for Wayland";
     homepage = "https://github.com/bugaevc/wl-clipboard";
+    description = "Command-line copy/paste utilities for Wayland";
     license = licenses.gpl3;
```

License seems to be [GPLv3+](https://github.com/bugaevc/wl-clipboard/blob/2c3cee1c8173d75b1411997b2c0f3e486194ac0c/src/util/misc.c#L5-L8).

```suggestion
    license = licenses.gpl3Plus;
```

---

I will add this as a new commit.